snap/snapcraft.yaml

```diff
@@ -0,0 +1,130 @@
+          name:                   sqlitebrowser  # the name of the snap
+          # version:                3.10.1  # the version of the snap
+          version: master
+          version-script:  printf "`cat currentrelease | head -n 1 `-` git -C . rev-parse --short HEAD`"
+
+          summary:                DB browser for Sqlite  # 79 char long summary
```

Comment:
Should probably be "DB Browser for SQLite". :smile:

Reply:
Sure i will make that change.